### DISCUSSION TITLE
Can I use supabase with temporalio?

### DISCUSSION THREAD

**Discussion by ctriley** [supabase/supabase#27261] [Questions]
Temporal.io uses postgresql as storage. I'm wondering if I can setup supabase to use as the datastore for temporal. The commands listed in their helm chart https://github.com/temporalio/helm-charts that would need to be run are:

```
# in https://github.com/temporalio/temporal git repo dir
export SQL_PLUGIN=postgres12
export SQL_HOST=postgresql_host
export SQL_PORT=5432
export SQL_USER=postgresql_user
export SQL_PASSWORD=postgresql_password

make temporal-sql-tool

./temporal-sql-tool --database temporal create-database
SQL_DATABASE=temporal ./temporal-sql-tool setup-schema -v 0.0
SQL_DATABASE=temporal ./temporal-sql-tool update -schema-dir schema/postgresql/v12/temporal/versioned

./temporal-sql-tool --database temporal_visibility create-database
SQL_DATABASE=temporal_visibility ./temporal-sql-tool setup-schema -v 0.0
SQL_DATABASE=temporal_visibility ./temporal-sql-tool update -schema-dir schema/postgresql/v12/visibility/versioned
```

The source code for this tool seems to be located at: https://github.com/temporalio/temporal/tree/main/tools/sql

**Comment by GaryAustin1** [comment]
Not commenting specifically on temporalio, but if it works with a Postgres database (and looks like the info it was is normal for that).
![image](https://github.com/supabase/supabase/assets/54564956/3bd5b9bb-10c8-4dd9-a5de-bda8529fbde7)

Then if you go to the database settings in the SB dashboard there will be the connection info for port 5432 and there is the connection page also:
![image](https://github.com/supabase/supabase/assets/54564956/1a7c9fa6-67fa-475d-8cf1-1c578875056e)
